src/metrics/metrics.jl

```diff
@@ -279,44 +158,39 @@ end
 #     return rs.outcomes[:relative_loc_taxa_cover]
 # end
 
-relative_loc_taxa_cover = Metric(_relative_loc_taxa_cover, (:timesteps, :taxa, :location, :scenarios))
+relative_loc_taxa_cover = Metric(_relative_loc_taxa_cover, (:timesteps, :taxa, :sites, :scenarios))
```

Comment:
I would have thought the dimension name `:location` or `:locations` was more general

Reply:
Yes, we are going to migrate from `:sites` to `:locations`, but that will be on a separate PR and I thought it would be better if all dimensions follow the same standard. Do you think this change (this one line in this PR, I mean) can cause any problem? Probably the migration to `:locations` will happen next (after migrating to YAXArrays).

Reply:
Ah ok, sounds good, I don't think it effects anything else